app/constants.py

```diff
@@ -0,0 +1,255 @@
+SMS_TYPE = 'sms'
+NOTIFICATION_TYPE = (EMAIL_TYPE, SMS_TYPE, LETTER_TYPE)
+
+# Notification status
+NOTIFICATION_CANCELLED = 'cancelled'
+NOTIFICATION_CREATED = 'created'
+NOTIFICATION_SENDING = 'sending'
+NOTIFICATION_SENT = 'sent'
+NOTIFICATION_DELIVERED = 'delivered'
+NOTIFICATION_PENDING = 'pending'
+NOTIFICATION_FAILED = 'failed'
+NOTIFICATION_TECHNICAL_FAILURE = 'technical-failure'
+NOTIFICATION_TEMPORARY_FAILURE = 'temporary-failure'
+NOTIFICATION_PERMANENT_FAILURE = 'permanent-failure'
+NOTIFICATION_PENDING_VIRUS_CHECK = 'pending-virus-check'
+NOTIFICATION_VALIDATION_FAILED = 'validation-failed'
+NOTIFICATION_VIRUS_SCAN_FAILED = 'virus-scan-failed'
+NOTIFICATION_RETURNED_LETTER = 'returned-letter'
+NOTIFICATION_CONTAINS_PII = 'pii-check-failed'
+NOTIFICATION_PREFERENCES_DECLINED = 'preferences-declined'
+
+NOTIFICATION_STATUS_TYPES = (
+    NOTIFICATION_CANCELLED,
+    NOTIFICATION_CREATED,
+    NOTIFICATION_SENDING,
+    NOTIFICATION_SENT,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_PENDING,
+    NOTIFICATION_FAILED,
+    NOTIFICATION_TECHNICAL_FAILURE,
+    NOTIFICATION_TEMPORARY_FAILURE,
+    NOTIFICATION_PERMANENT_FAILURE,
+    NOTIFICATION_PENDING_VIRUS_CHECK,
+    NOTIFICATION_VALIDATION_FAILED,
+    NOTIFICATION_VIRUS_SCAN_FAILED,
+    NOTIFICATION_RETURNED_LETTER,
+    NOTIFICATION_CONTAINS_PII,
+    NOTIFICATION_PREFERENCES_DECLINED,
+)
+
+NOTIFICATION_STATUS_TYPES_COMPLETED = (
+    NOTIFICATION_SENT,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_FAILED,
+    NOTIFICATION_TECHNICAL_FAILURE,
+    NOTIFICATION_TEMPORARY_FAILURE,
+    NOTIFICATION_PERMANENT_FAILURE,
+    NOTIFICATION_RETURNED_LETTER,
+    NOTIFICATION_CANCELLED,
+    NOTIFICATION_PREFERENCES_DECLINED,
+)
+
+NOTIFICATION_STATUS_TYPES_FAILED = (
+    NOTIFICATION_TECHNICAL_FAILURE,
+    NOTIFICATION_TEMPORARY_FAILURE,
+    NOTIFICATION_PERMANENT_FAILURE,
+    NOTIFICATION_VALIDATION_FAILED,
+    NOTIFICATION_VIRUS_SCAN_FAILED,
+    NOTIFICATION_RETURNED_LETTER,
+    NOTIFICATION_CONTAINS_PII,
+    NOTIFICATION_PREFERENCES_DECLINED,
+)
+
+NOTIFICATION_STATUS_TYPES_BILLABLE_FOR_LETTERS = (
+    NOTIFICATION_SENDING,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_RETURNED_LETTER,
+)
+
+NOTIFICATION_STATUS_TYPES_BILLABLE = (
+    NOTIFICATION_SENDING,
+    NOTIFICATION_SENT,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_FAILED,
+    NOTIFICATION_TEMPORARY_FAILURE,
+    NOTIFICATION_PERMANENT_FAILURE,
+    NOTIFICATION_RETURNED_LETTER,
+)
+
+# VA Profile types
+MOBILE_TYPE = 'mobile'
+
+# Template
+TEMPLATE_TYPES = (SMS_TYPE, EMAIL_TYPE, LETTER_TYPE)
+TEMPLATE_PROCESS_NORMAL = 'normal'
+TEMPLATE_PROCESS_PRIORITY = 'priority'
+TEMPLATE_PROCESS_TYPE = (TEMPLATE_PROCESS_NORMAL, TEMPLATE_PROCESS_PRIORITY)
+
+# Callbacks
+DELIVERY_STATUS_CALLBACK_TYPE = 'delivery_status'
+COMPLAINT_CALLBACK_TYPE = 'complaint'
+INBOUND_SMS_CALLBACK_TYPE = 'inbound_sms'
+SERVICE_CALLBACK_TYPES = (DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE, INBOUND_SMS_CALLBACK_TYPE)
```

Comment:
These two are only used in validation schema and the validation error raised used string interpolation to add the tuple/list to the validation error, so if we added or removed any it'd also break. Since it's only used in one spot on an error I cast the tuple to a list to maintain backwards compatibility. Also walked ALL validation errors raised in the code, looking for any more and none were found (other than the channels you pointed out).